lumigator/python/mzai/backend/backend/ray_submit/submission.py

```diff
@@ -37,6 +38,11 @@ def submit_ray_job(client: JobSubmissionClient, entrypoint: RayJobEntrypoint) ->
     loguru.logger.info(
         f"Submitting {entrypoint.get_command_with_params}...{entrypoint.runtime_env}"
     )
+    # copy the lumigator_schema to the working directory so that it's uploaded with the source code
+    # This way, the jobs can use and adhere to the Lumigator schemas when running in ray.
+    working_dir = entrypoint.runtime_env["working_dir"]
+    os.system(f"cp -r ../schemas/lumigator_schemas/ {working_dir}")
```

Comment:
I'm open to thoughts about how else you would add lumigator_schemas to a job. I thought about adding it to the requirements.txt, but then you would need to publish it or somehow register it on the cluster.

Reply:
The schemas are (in the process of being) actually published to PyPI. However, I do agree on sharing them internally. Note that we are not adding jobs dynamically, so I'd do this kind of packaging (bundling it into a zip too might be worth the effort) once during startup.

Maybe adding a venv with the dependencies too is a possible strategy, but @veekaybee, if I got that right, you already explored this in lm_buddy and were not satisfied with the result?

Reply:
yep, see here https://github.com/mozilla-ai/lumigator/pull/692#issuecomment-2604888575